bundles/fluent-en_GB/atrium-fluent-en_GB-jvm/build.gradle

```diff
@@ -10,4 +10,10 @@ dependencies {
 
     runtimeOnly prefixedProject('domain-robstoll-jvm')
     runtimeOnly prefixedProject('core-robstoll-jvm')
+
+    //spek2 dependencies configuration
+    runtimeOnly "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    runtimeOnly "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+    api "org.spekframework.spek2:spek-dsl-jvm:$spek2_version", excludeKotlin
+    runtimeOnly "org.spekframework.spek2:spek-runner-junit5:$spek2_version", excludeKotlin
```

Comment:
This should not be required as it is already setup elsewhere

Reply:
When removing these lines I get several errors during execution of gr build command.
Errors like "unresolved reference" java.lang.NoClassDefFoundError 

Probably some sort of api prefixedProject is required but I couldnt figure out which exactly should be used

Can you direct me here?

Reply:
I had a quick look and I was wrong, spek2 is not added to the smoke-test modules. Nevertheless the dependencies should not be added to runtimeOnly (because this way we would distribute Spek with Atrium and we only want to use spek for tests).

I have fixed this in master. Please rebase your branch (see https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#git in case you have never done this before).

Once you rebased you can remove the above lines.

Reply:
Done, committed

Reply:
I actually meant rebase and not merge :wink: But also fine, I will just squash your commit in this case